static/app/components/organizations/pageFilters/utils.tsx

```diff
@@ -136,15 +140,24 @@ export function getPageFilterStorage(orgSlug: string) {
     return null;
   }
 
+  let decoded: any;
+
   try {
-    return JSON.parse(value) as Omit<PageFilters, 'datetime'>;
+    decoded = JSON.parse(value);
   } catch (err) {
     // use default if invalid
     Sentry.captureException(err);
     console.error(err); // eslint-disable-line no-console
+
+    return null;
   }
 
-  return null;
+  const result: RetrievedData = {
```

Comment:
was not it better to keep this code inside the try?

Reply:
The code didn't do anything before. Now it correct coerces things